src/routes/userRouter.ts

```diff
@@ -0,0 +1,15 @@
+
+import express from "express";
+
+// Import controller
+import userControllers from "../modules/user/controller/userControllers";
+
+// Import Validations
+import {validateUpdateUserRole} from "../middlewares/validation"
+
+const userRouter = express.Router();
+
+// Update the users role
+userRouter.put("/update-role/:id",validateUpdateUserRole, userControllers.updateUserRole);
```

Comment:
Please follow the code convention we are using in our project. Firstly, we are not using Joi Validations in this way.